Server/Settings.cs

```diff
@@ -28,7 +28,8 @@ private static MessageQueue MessageQueue
             NameListPath = Path.Combine(EnvirPath, "NameLists"),
             ValuePath = Path.Combine(EnvirPath, "Values"),
             NoticePath = Path.Combine(EnvirPath, "Notice.txt"),
-            MinimapsPath = Path.Combine(EnvirPath, "Minimaps");
+            MinimapsPath = Path.Combine(EnvirPath, "Minimaps"),
+            NPCPreviewPath = Path.Combine(EnvirPath, "NPCPreviewPath");
```

Comment:
Can you move this and the minimap path in to a sub folder of "Previews". And also rename "NPCPreviewPath" to just "NPCs"

Reply:
Yea can do.